aspnetcore/performance/caching/hybrid.md

```diff
@@ -230,13 +231,24 @@ dotnet add package Microsoft.Extensions.Caching.SqlServer
 
 A concrete implementation of the `HybridCache` abstract class is included in the shared framework and is provided via dependency injection. But developers are welcome to provide or consume custom implementations of the API, for example [FusionCache](https://github.com/ZiggyCreatures/FusionCache/blob/main/docs/MicrosoftHybridCache.md).
 
+## Use Hybrid Cache with Native AOT
+
+ The following Native AOT-specific considerations apply to `HybridCache`:
+
+* **Serialization**  
+
+  Native AOT doesn't support runtime reflection-based serialization. If you cache custom types, you must use source generators or explicitly configure serializers that are compatible with AOT, like `System.Text.Json` source generation. For example, with `System.Text.Json`, register a `JsonSerializerContext` for your types and configure it in `AddHybridCache`.
```

Comment:
This refers to a future / proposed API addition;  *at the moment*, this guidance would be very unclear. Question (genuinely): which is better:

1. issue guidance today that is obscure but tells them how to configure HybridCache for AOT via DI
2. defer until this is neatly wrapped in a managed easy to use API

?

Reply:
@mgravell 

> This refers to a future / proposed API addition; _at the moment_, this guidance would be very unclear. Question (genuinely): which is better:
> 
> 1. issue guidance today that is obscure but tells them how to configure HybridCache for AOT via DI
> 2. defer until this is neatly wrapped in a managed easy to use API
> 
> ?

I would go for option two and remove the "For example" sentence. In place of that, say
> `HybridCache` is still under development, and simplifying the way to use it with AOT is a high priority for that development.